packages/itwinui-css/src/avatar/avatar.scss

```diff
@@ -3,59 +3,39 @@
 @use '../mixins';
 @use './mixins' as *;
 
+$avatar-text-color: hsl(0 0% 0% / 0.6); // same as badge, meets AA contrast against soft backgrounds
+
 .iui-avatar {
   @include mixins.iui-reset;
   display: inline-flex;
   align-items: center;
   justify-content: center;
   position: relative;
-  margin: 0;
-  user-select: none;
   inline-size: var(--_iui-avatar-size);
   block-size: var(--_iui-avatar-size);
-  background-color: hsl(72, 51%, 56%);
+  background-color: var(--iui-color-background-seagreen);
```

Comment:
I don't see `--iui-color-background-seagreen` defined anywhere.

Since all avatars have a color assigned based off the initials, how about we set the default color to `var(--iui-color-background)` and remove:
```scss
.iui-avatar-count {
  background-color: var(--iui-color-background);
}
```

https://github.com/iTwin/iTwinUI/blob/648b4ca22976b0de7bfa892ddcf31dd930f309c2/packages/itwinui-css/src/avatar/avatar.scss#L16C8-L16C8

Reply:
Ahh right, I forgot about hardcoding the text color so using `--iui-color-background` would not work.  Your change sounds good to me.